Libraries/libs.mk

```diff
@@ -138,8 +138,14 @@ ifeq ($(LIB_SDHC), 1)
 # Set the SDHC driver directory
 SDHC_DRIVER_DIR ?= $(LIBS_DIR)/SDHC
 
+# Set FatFS version.  Acceptable values:
+# - ff13
+# - ff14
+# - ff15
+FATFS_VERSION ?= ff15
```

Comment:
Should we keep the default at ff13 so legacy builds don't break and some deprecated warnings before eventually switching to ff15 as the default?

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/720/commits/ebbf3283180f34055ba5a81df63be683746d9a0e